android/app/src/main/java/com/google/codeu/chatme/model/Conversation.java

```diff
@@ -51,6 +52,15 @@ public void setTimeCreated(long timeCreated) {
         this.timeCreated = timeCreated;
     }
 
+    public String getLastMessageContent() {
+        return this.lastMessage.getContent();
+    }
+
+    public String getReadableLastMessageTime() {
+        // TODO: format string
```

Comment:
Haha! Well, `System.currentTimeMillis()` gets us time in milliseconds elapsed since the [epoch](https://en.wikipedia.org/wiki/Epoch_(reference_date)#Computing). It's better to store time in this form to facilitate sorting. I will add a function to convert this `long` into human-readable format.